addons/web/static/src/js/widgets/rainbow_man.js

```diff
@@ -0,0 +1,62 @@
+odoo.define('web.rainbow_man', function (require) {
+"use strict";
+
+var ajax = require("web.ajax");
+var Widget = require('web.Widget');
+var core = require('web.core');
+var qweb = core.qweb;
+var _t = core._t;
+
+ajax.loadXML("/web/static/src/xml/rainbow_man.xml", qweb);
```

Comment:
ok, but this is not a good method.  With this, a RPC will be done each time someone loads the backend, even if it is useless in that case.  I propose to instead load only the file in the willStart method of the widget, and only if it is needed (see below)